test/elixir_analyzer/test_suite/high_school_sweetheart_test.exs

```diff
@@ -0,0 +1,116 @@
+defmodule ElixirAnalyzer.TestSuite.HighSchoolSweetheartTest do
+  alias ElixirAnalyzer.Constants
+
+  use ElixirAnalyzer.ExerciseTestCase,
+    exercise_test_module: ElixirAnalyzer.TestSuite.HighSchoolSweetheart
+
+  test_exercise_analysis "example solution",
+    comments: [] do
+    defmodule HighSchoolSweetheart do
```

Comment:
Is this the exemplar solution? If yes, since #157 you would need to add one more comment like
> https://github.com/exercism/elixir-analyzer/blob/ca5fd50ff10ba572186690473d13b2befd9ccc41/test/elixir_analyzer/test_suite/bird_count_test.exs#L6

Reply:
Ah yes, I had a look. I think the problem is since the code is inside of a block, the indentation of the multiline strings is all messed up. There is a fix, you can pass the code as a string in this way:

```elixir
  test_exercise_analysis "example solution",
    comments: [ElixirAnalyzer.Constants.solution_same_as_exemplar()] do
    ~S'''
    defmodule HighSchoolSweetheart do
    ...
    end
    '''
  end
```
If the the input is a string, it will get parsed as an AST straight way anyway, but at least in this way you're in control of the indentation. 
Multiline strings are a pain to deal with, we had several hardships related to that :)

Reply:
You also need to run `mix format` on this file.

Reply:
It worked!! thanks, I tried almost everything :laughing:

Reply:
Believe me, I spent hours over weeks on something like this, I even found a bug in the language (that was already patched in Elixir `main` but still), I am now an Elixir multiline pro debugger.